shared-bindings/wifi/__init__.c

```diff
@@ -43,6 +67,9 @@ static const mp_rom_map_elem_t wifi_module_globals_table[] = {
 
     // Properties
     { MP_ROM_QSTR(MP_QSTR_radio),       MP_ROM_PTR(&common_hal_wifi_radio_obj) },
+
+    { MP_ROM_QSTR(MP_QSTR_supports_ipv6), MP_ROM_PTR(CIRCUITPY_SOCKETPOOL_IPV6 ? MP_ROM_TRUE : MP_ROM_FALSE) },
+    { MP_ROM_QSTR(MP_QSTR_supports_ipv4), MP_ROM_TRUE },
```

Comment:
How would you determine this in CPython? Should it be on Radio or SocketPool (if CPython's `socket` has a way to detect it.) I'm thinking ahead to having a thread network interface that is ipv6-only (I think.)

Reply:
good question. I don't think there's a look-before-you-leap "is there working ipv6" check in CPython.. However, in CPython with no ipv6 global connectivity, a v6 connect() to an internet address will promptly fail:
```python
>>> s.connect(("ipv6.google.com", 80))
OSError: [Errno 101] Network is unreachable
```

If the OS lacked even support compile-time for V6 sockets at all then the socket module can lack the AF_INET6 definition:
```c
#ifdef AF_INET6
    ADD_INT_MACRO(m, AF_INET6); /* IP version 6 */
#endif
```
and creating a socket with an unsupported family will raise an exception:
```python
>>> socket.socket(socket.AF_UNSPEC, socket.SOCK_STREAM)
OSError: [Errno 97] Address family not supported by protocol
```

Reply:
I can drop these properties if they're not useful. I really only added them so that the documentation could show a "correct" default parameter for `start_dhcp_server`, by showing the constant `supports_ipv6` instead of a literal `True` or `False` which would be incorrect

Reply:
`OSError: [Errno 97] Address family not supported by protocol` makes sense to me. One can always catch it.

Reply:
> I can drop these properties if they're not useful. I really only added them so that the documentation could show a "correct" default parameter for `start_dhcp_server`, by showing the constant `supports_ipv6` instead of a literal `True` or `False` which would be incorrect

I wouldn't change the default based on functionality. Code will run different on different platforms then.